components/uuf-core/src/main/java/org/wso2/carbon/uuf/internal/deployment/OsgiPluginProvider.java

```diff
@@ -37,24 +39,27 @@
            service = PluginProvider.class,
            immediate = true,
            property = {
-                   "componentName=wso2-uuf-server"
+                   "componentName=wso2-uuf-OSGi-plugin-provider"
            }
 )
 public class OsgiPluginProvider implements PluginProvider {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(OsgiPluginProvider.class);
     private final Map<Class, ServiceTracker> serviceTrackers = new HashMap<>();
     private BundleContext bundleContext = null;
 
     @Activate
     protected void activate(BundleContext bundleContext) {
         this.bundleContext = bundleContext;
+        LOGGER.debug("OsgiPluginProvider activated.");
```

Comment:
Since we use SLF4J, AFAIK we don't need to check the `isDebugEnabled` (refer [org.ops4j.pax.logging.slf4j.Slf4jLogger](https://github.com/ops4j/org.ops4j.pax.logging/blob/logging-1.8.4/pax-logging-api/src/main/java/org/ops4j/pax/logging/slf4j/Slf4jLogger.java#L272-L275)). Also this is a simple log, putting the if check might be an overhead (see [blog](https://www.igorkromin.net/index.php/2015/05/12/are-guard-statements-like-isdebugenabled-necessary-when-using-log4j/))